code/datums/entities/player_misc_stat.dm

```diff
@@ -0,0 +1,26 @@
+/datum/entity/player_misc_stat
+	var/player_id
+	var/stat_id
+	var/stat_number
+
+BSQL_PROTECT_DATUM(/datum/entity/player_misc_stat)
+
+/datum/entity_meta/player_misc_stat
+	entity_type = /datum/entity/player_misc_stat
+	table_name = "player_misc_stat"
```

Comment:
rename this to player_stats, or something similar. Avoid words like 'misc' in titles because it's not detailed enough.

Consider adding a 'stat_category' to be able to more easily differentiate between statistics stored in this table.